fetch.bs

```diff
@@ -2436,7 +2436,9 @@ with a <i>CORS flag</i> and <i>recursive flag</i>, run these steps:
 
   <dl class=switch>
    <dt><var>request</var>'s <a for=request>current url</a>'s
-   <a for=url>origin</a> is <var>request</var>'s
+   <a for=url>origin</a> is the
+   <a href="https://html.spec.whatwg.org/multipage/#same-origin">same</a>
```

Comment:
You can change this to `<a>same origin</a>`with phrasing "current url's origin is [same origin] with request's origin". That will end up linking to HTML's definition once generated.